src/ol/interaction/KeyboardZoom.js

```diff
@@ -41,7 +46,15 @@ class KeyboardZoom extends Interaction {
      * @private
      * @type {import("../events/condition.js").Condition}
      */
-    this.condition_ = options.condition ? options.condition : targetNotEditable;
+    this.condition_ = options.condition
+      ? options.condition
+      : function (mapBrowserEvent) {
+          return (
+            (noModifierKeys(mapBrowserEvent) ||
+              shiftKeyOnly(mapBrowserEvent)) &&
+            targetNotEditable(mapBrowserEvent)
+          );
+        };
```

Comment:
Why not simply
```suggestion
      : (mapBrowserEvent) => !platformModifierKeyOnly(mapBrowserEvent) && targetNotEditable(mapBrowserEvent)
```
?

Reply:
It will not respond to + on keyboards which need shift to type +.

Reply:
(although on Windows Chrome with UK keyboard ctrl = will also activate browser zoom).